### PR TITLE
Add `get_wrapper` to prevent 429 errors to prevent scraping

### PR DESCRIPTION
**Pull request by vishaalagartha**
[What]
Introduce `get_wrapper` to handle when there is a Retry-After in the response header. Allows scraping to continue after 429 errors

[How]
Add `sleep` functionality to pause execution.

### CODE REVIEW

**Comment by vishaalagartha** [comment]
@amywinecoff can you review this PR and see if this fix would resolve the 429 errors?

File: basketball_reference_scraper/request_utils.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from requests import get
+from time import sleep
+
+def get_wrapper(url):
+    r = get(url)
+    while True:
```
Comment (tgracin) [inline basketball_reference_scraper/request_utils.py:6]: simpler variant of what you wrote: `while r.status_code != 200`
but this way you cannot return anything that is not 200, and catch errors.
so, probably best:
`while r.status_code == 429: ...`

File: basketball_reference_scraper/seasons.py [hunk 2]
```diff
@@ -3,6 +3,12 @@
 from requests import get
```
Comment (tgracin) [inline basketball_reference_scraper/seasons.py:3]: you'd probably want to remove this

**Comment by tgracin** [comment]
I did it a bit differently (I maintained session and used exponential backoff), but your version with retries after the suggested time from header works as well.

My commit: https://github.com/vishaalagartha/basketball_reference_scraper/commit/830f76a9999e613b4138eb376387df3f1b52428a

**Pull request merged**